### DISCUSSION TITLE
🎬 documentation / tutorials: List of planned or requested video tutorials

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#2081] [users: ideas for new features]
We'll collect wishes and ideas for video tutorials here.
Please contribute.

@arch-kiosk/test

**Comment by urapadmin** [comment]
- [x] Using Touch Draw and FileMaker recording on the iPad
- [x] how to use file picking (for testers and bold admins): #2760 
- [x] what file picking can do: #2759 
- [x] #2096 
- [x] how to install a Kiosk patch

**Comment by luizaogs** [comment]
How to navigate the basics of FM configuration? Or is that silly and is the expectation that field directors will not have to do so? And I guess with that I mean both simple config but also things like identifier lists, adding options to drop downs, etc.

Different types of image import in the file repository?

How to use the registration module efficiently?

**Reply by urapadmin** [reply]
rather sooner than later we need this:
- [ ] #2135

**Reply by luizaogs** [reply]
I do think the registration tutorial is necessary, too. I was reminded of how I still do not know how to use it properly this morning and can imagine that it would be overwhelming at first for someone who is confronted with all of the available buttons and filters, etc.

**Reply by urapadmin** [reply]
- [x] #2151

**Comment by luizaogs** [comment]
Oh also, object photography? Though if e-ink is substituting that soonish maybe that would be a waste of your time.

**Reply by urapadmin** [reply]
unlikely soonish. First somebody with 3D printing and modelling experience needs to come up with a smart encasing for the electronics.

**Reply by urapadmin** [reply]
so that would be one for @lbestock if she can keep it short

**Reply by urapadmin** [reply]
Perhaps it makes sense to do that when #1977 is implemented.

**Comment by urapadmin** [comment]
- [x] list of new features since Fall 2022 (so everything that has changed since the last version in the field): #2094 
- #2092

**Comment by urapadmin** [comment]
- [x] #2097 
- [ ] Where does my Kiosk data live?
- [ ] export data and use it in MS Excel
- [ ] export data and use it in Microsoft Access
- [ ] #2825 
- [ ] tutorial: How to use Kiosk Data in GIS

**Comment by urapadmin** [comment]
- [ ] #2107

**Comment by lbestock** [comment]
- [x]  How to use the File Repository (for instance, when looking for images for a report)

**Comment by urapadmin** [comment]
- [x] #2137

**Comment by lbestock** [comment]
- [x] #2126

**Comment by lbestock** [comment]
- [x] Daily admin with multiple iPads using Kiosk Bridge

**Reply by urapadmin** [reply]
honestly, I think showing it with a single ipad would suffice. I am still baffled.

**Comment by urapadmin** [comment]
- [ ] #2136

**Comment by urapadmin** [comment]
- [x] #2125

**Comment by lbestock** [comment]
- [ ] Quality Control, what it is, what its effects are in the trench and in Director's View. Possibly how to set it up?

**Reply by urapadmin** [reply]
how to set it up is currently for cutting edge admins. I am not sure we should go that far. More: How to get it (ask us).

**Comment by lbestock** [comment]
- [ ] #2761 
- [ ] #2762

**Comment by luizaogs** [comment]
Should directors/admins know how to generate different reports? Or is that you, Lutz? Because seeing a bunch of reporting tickets in testing makes me realize I have no idea how to deal with reporting. So potentially something that needs a tutorial.

- [x] #2168

**Reply by urapadmin** [reply]
it is me in the sense that I install the reports on their systems and they just run and download them. But you are right, I think a tutorial on how to install and run a report could not hurt.
More important, though: how to install / update a report should be in the Q&A.

**Reply by urapadmin** [reply]
Since I had to install the PASU report, it was a good occasion to make that video.

**Reply by luizaogs** [reply]
Oh yay, so now I can test report stuff :)

**Reply by urapadmin** [reply]
not that you needed that tutorial to do so :-}

**Comment by urapadmin** [comment]
@lbestock 
### 2024 features tutorial
- quick overview of Q&V. Don't forget charts and export to CSV/Excel. And then there is #2831 
- quick overview of Stratigraphy Studio
- Harris Matrix in recording app 
- recording app: sortable headers for collected material lists everywhere
- ✏️ smaller sequence qr codes for object photography and easier focus
- 🌷 Query And View has a new query "Full Text Search"
- Month and day get so easily confused in date fields, particularly when inputting legacy data. That's why Kiosk has always shown a Latin representation of your input. NOW it allows you to enter dates in the Latin format almost everywhere where you find a date / date time field. You can even type something short like 2IV89 to get April 2nd 1989 (https://github.com/arch-kiosk/arch-kiosk-office/issues/2416)
- ❄️ For units/trenches you can record a list of involved people (they don't even have to be team members) and their roles. So far there was only a supervisor. (https://github.com/arch-kiosk/arch-kiosk-office/issues/2415)
- 🌻 You can now tag a candid in the file repository as "map" and it will get transported to the recording app's gallery (https://github.com/arch-kiosk/arch-kiosk-office/issues/2288,https://github.com/arch-kiosk/arch-kiosk-office/issues/2278)

⌚ manage todos with the recording app (#2414)

⌚ create new context while recording stratigraphy (#2908)

⌚ improved context selector in context relations (#2961)

photos in narrative/photos now are easily visually distinguished as having been linked

- Kiosk with time zones 

anything I forgot? @luizaogs, do you think something else needs mentioned?

**Reply by luizaogs** [reply]
Seems pretty complete to me.

**Comment by lbestock** [comment]
Weirdly enough, I am wanting a teeny video on using the "answers to questions" guides.

**Reply by luizaogs** [reply]
When I saw the new website organization I wondered if doing something similar for the q&as would be helpful. There are so many questions that it's hard to find stuff even if one knows what they're looking for. The headings wouldn't necessarily be record, study, organize, but I don't know if breaking up the ToC into separate sections would be helpful.
